packages/ember-cli-fastboot/addon/locations/none.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable ember/no-classic-classes, ember/no-get, ember/require-computed-property-dependencies, prettier/prettier */
 import { computed, get } from '@ember/object';
 import { bool, readOnly } from '@ember/object/computed';
-import { inject as service } from '@ember/service';
+import { service } from '@ember/service';
```

Comment:
@felixbuenemann for https://github.com/ember-fastboot/ember-cli-fastboot/pull/955#issuecomment-3511131216 I think it goes like this

```suggestion
import serviceModule from '@ember/service';

const service = serviceModule.service ?? serviceModule.inject;
```

Reply:
Thank you, it must be `import * as serviceModule from '@ember/service'` but then it works.